### PR TITLE
Multiarch builds - acas-roo-server

### LINKED ISSUE #281: Add multi-arch builds - acas-roo-server

### PR DESCRIPTION
**Pull request by brianbolt**
Fixes #281

### CODE REVIEW

File: Dockerfile-bbchem [hunk 1]
```diff
@@ -15,17 +15,48 @@ RUN 	mvn dependency:resolve -P ${CHEMISTRY_PACKAGE}
 ADD 	. /src
 RUN 	mvn clean && \
         mvn compile war:war -P ${CHEMISTRY_PACKAGE}
-RUN     mv target/acas*.war $CATALINA_HOME/webapps/acas.war && \
-        mv target/acas* $CATALINA_HOME/webapps/acas/
 
-FROM 	${TOMCAT_IMAGE} as build
-COPY 	--chown=runner:runner --from=compile /usr/local/tomcat/webapps/acas.war $CATALINA_HOME/webapps/acas.war
-COPY 	--chown=runner:runner --from=compile /usr/local/tomcat/webapps/acas $CATALINA_HOME/webapps/acas
+FROM tomcat:9.0.58-jre8-openjdk-slim-buster
```
Comment (brianbolt) [inline Dockerfile-bbchem:19]: Most of the stuff below here is stuff that was in `mcneilco/tomcat-maven` repo previously.

You'll also see this repeated in most of the docker files that we are keeping around to support chef based docker builds which cannot use multi-stage builds.  Eventually we'll only keep one, multistage, Dockerfile

File: pom.xml [hunk 2]
```diff
@@ -46,7 +46,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.]]>
 		<cdk.version>1.5.13</cdk.version>
 		<jchem.version>16.4.25.0</jchem.version>
 		<rdkit.version>2021.03.5-release</rdkit.version>
-		<indigo.version>1.2.3.r0</indigo.version>
+		<indigo.version>1.6.1</indigo.version>
```
Comment (brianbolt) [inline pom.xml:49]: Had to bump this in order to compile on arm64

File: wait-for-it.sh [hunk 3]
```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/env bash
```
Comment (brianbolt) [inline wait-for-it.sh:1]: This was taken directly from the tomcat-maven repo

**Review by bffrost — APPROVED** [review]
LGTM

**Review by bffrost — APPROVED** [review]
LGTM. This makes more sense. Just tagging on branch, pr, or tag, and just appending chemistry package

**Pull request merged**